netlify.toml

```diff
@@ -0,0 +1,4 @@
+[build]
+  base    = "docs/"
+  publish = "public/"
+  command = "gatsby build --prefix-paths && mkdir -p docs/apollo-cli && mv public/* docs/apollo-cli && mv docs public/ && mv public/docs/apollo-cli/_redirects public"
```

Comment:
maybe we can make this an npm script "build-deploy"

Reply:
I personally would rather leave this as-is to keep parity with all the other docs in other repos, and I think at least the `_redirects` stuff is specific to netlify